internal/cli/opsmanager/metrics/disks_describe.go

```diff
@@ -39,14 +39,19 @@ func (opts *DisksDescribeOpts) initStore() error {
 	return err
 }
 
+var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurements.Measurements}}
+{{.Name}}	{{.Units}}{{range .DataPoints}}	
+		{{.Timestamp}}	{{.Value}}{{end}}{{end}}
```

Comment:
I left this for last as I knew it was going to be a challenge to make sense of it, and I was right, in this current format it's really hard to understand what's going on, this template may need some extra logic to it, here a couple of points for it (I'm still learning how templates work so I may type errors here)

You can use variables and `if` conditions so I'd say we need to do those here with something like
```suggestion
var diskMetricTemplate = `NAME	UNITS	TIMESTAMP		VALUE{{range .ProcessMeasurements.Measurements}} {{ $name = .Name}}	{{$unit = .Units}}
{{- if .DataPoints }}}{{range .DataPoints}}	
 {{ $name }} {{$unit}} {{.Timestamp}}	{{.Value}}{{end}}{{end}}{{end}}
```
This way each data point always has a name and a unit, we also make sure that if there's no data point we don't print anything (if of empty slices is false on templates)